### PR TITLE
Added Getting Started for Service Registry

### PR DESCRIPTION
**Pull request by wtrocki**
PR restoring and covering work done in https://github.com/redhat-developer/app-services-guides/pull/288

@bibryam @smccarthy-ie feel free to continue working on this

### CODE REVIEW

File: getting-started-service-registry/quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: getting-started-service-registry
+  draft: true
```
Comment (wtrocki) [inline getting-started-service-registry/quickstart.yml:5]: wrong place. should be in annotations

File: getting-started-service-registry/README.adoc [hunk 2]
```diff
@@ -0,0 +1,197 @@
+[id="chap-getting-started-service-registry"]
+= Getting Started with {product-long}
+ifdef::context[:parent-context: {context}]
+:context: getting-started-service-registry
+
+////
+START GENERATED ATTRIBUTES
```
Comment (wtrocki) [inline getting-started-service-registry/README.adoc:7]: I think this was copied from other place. We need to regenerate it

File: getting-started-service-registry/quickstart.yml (hunk 1 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/quickstart.yml:5]: Looks like the contrib guide is incorrect? 
https://github.com/redhat-developer/app-services-guides/blob/main/CONTRIBUTING.adoc#draft-quick-starts

File: getting-started-service-registry/README.adoc (hunk 2 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/README.adoc:7]: Yes I think so too, will commit with regenerated attributes.

File: getting-started-service-registry/quickstart.yml (hunk 1 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/quickstart.yml]: fixed .quickstart.yml (and contrib guide) in latest commit

File: getting-started-service-registry/README.adoc (hunk 2 above)
Comment (smccarthy-ie) [reply getting-started-service-registry/README.adoc:2]: done

**Comment by smccarthy-ie** [comment]
@wtrocki @pmuir @bibryam @EricWittmann @sterobin 
I've cleaned up this PR and tested locally so all checks are passing :) 

Please review and comment if any more changes are needed. Thanks!

**Review by pmuir — APPROVED** [review]

**Comment by bibryam** [comment]
@smccarthy-ie  LGTM

**Pull request merged**